test/integration/setup/network.js

```diff
@@ -14,11 +14,26 @@
 
 'use strict';
 
-var async = require('async');
-var Promise = require('bluebird');
-var waitUntilBlockchainReady = require('../../common/utils/wait_for')
+const async = require('async');
+const Promise = require('bluebird');
+const waitUntilBlockchainReady = require('../../common/utils/wait_for')
 	.blockchainReady;
-var utils = require('../utils');
+const utils = require('../utils');
+
+const SYNC_MODES = {
```

Comment:
Would be nice to add comments what those constants are for.

Reply:
Those variables are barely used. It is coming from @MaciejBaj. Perhaps he can give us more input.